.tests/technitium-logs/config.yaml

```diff
@@ -0,0 +1,9 @@
+parsers:
+  - ./parsers/s00-raw/crowdsecurity/syslog-logs.yaml
```

Comment:
change this to `crowdsecurity/syslog-logs`

and add `crowdsecurity/dateparse-enrich`